src/czbenchmarks/tasks/single_cell/perturbation_expression_prediction.py

```diff
@@ -88,6 +109,7 @@ def __init__(
         self,
         metric: str = "wilcoxon",
         control_prefix: str = "non-targeting",
```

Comment:
Please use nomenclature and default value from the dataset class:

`control_name: str = "ctrl"`

https://github.com/chanzuckerberg/cz-benchmarks/blob/main/src/czbenchmarks/datasets/single_cell_perturbation.py#L92

Reply:
Yes, you're correct. I may have changed this and forgotten.. I'll close this for now. My apologies for the mistake.